LINK/usr/bin/miq_postgres_upgrade.sh

```diff
@@ -71,8 +71,11 @@ EOS
 read -p "Are you sure you want to proceed? (Y/N): " response
 if [ "$response" != "Y" ]
 then
+  if [ "$response" != "y" ]
```

Comment:
Maybe just add this to the conditional above?

`if [ "$response" != "Y" -a "$response" != "y" ]` should do the trick.

Then we don't need the extra if-block.